### DISCUSSION TITLE
guide/drivers/Trainbit

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#270] [Announcements]
# guide/drivers/Trainbit

Trainbit official website：https://trainbit.com/ AUSHELLPORTAL ，Apikey After entering the Trainbit webpage, press F12 to open the developer mode, open any refreshed data in the n...

https://alist.nn.ci/guide/drivers/Trainbit.html

**Comment by happybigboy** [comment]
this site will be off on 1 November, pls Delete the site
